### PR TITLE
Memmap: More error checking for Memset

### PR DESCRIPTION
**Pull request by JosJuice**
It now uses the same error checking as CopyFromEmu and CopyToEmu.

### CODE REVIEW

**Comment by degasus** [comment]
What do you think about such a function?
```
void* GetValidPointer(u32 addr, u32 length);
```

**Comment by JosJuice** [comment]
Sure, I can change GetValidPointer to that. It doesn't matter much, but it'll save a call to GetPointer.

**Comment by phire** [comment]
LGTM (assuming buildbot passes)

File: Source/Core/Core/HW/Memmap.cpp [hunk 1]
```diff
@@ -244,11 +244,18 @@ bool AreMemoryBreakpointsActivated()
 #endif
 }
 
-static inline bool ValidCopyRange(u32 address, size_t size)
+static inline void* GetPointerForRange(u32 address, size_t size)
```
Comment (lioncash) [inline Source/Core/Core/HW/Memmap.cpp]: Please don't use void as a return type. Use u8* like what GetPointer returns.

**Comment by lioncash** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**